src/acvp_aes.c

```diff
@@ -733,7 +733,7 @@ static ACVP_RESULT acvp_aes_output_tc (ACVP_CTX *ctx, ACVP_SYM_CIPHER_TC *stc,
          * Only return IV on AES-GCM ciphers
          */
         if (stc->cipher == ACVP_AES_GCM) {
-            rv = acvp_bin_to_hexstr(stc->iv, stc->iv_len, (unsigned char *) tmp);
+            rv = acvp_bin_to_hexstr(stc->iv, stc->iv_len, tmp, ACVP_SYM_CT_MAX);
```

Comment:
Related to this, I think the calloc above needs +1 for terminator (even though you never changed that code :) )